<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiGroupWidget</class>
 <widget class="QWidget" name="MultiGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="stateLabel1">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="settingBtn1">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="groupLabel3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr3:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="settingComboBox1">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="settingBtn2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="stateLabel2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="groupNoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr NO.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="groupLabel1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr1:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="groupLabel2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr2:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="groupLabel4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr4:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="settingBtn3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="groupLabel5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr5:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="groupLabel6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr6:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="stateLabel3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="groupLabel7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr7:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="settingComboBox2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="groupLabel8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gr8:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="settingComboBox4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="settingComboBox5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="settingComboBox6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="settingComboBox7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="settingComboBox8">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="settingBtn4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="settingBtn5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="settingBtn6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="settingBtn7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="settingBtn8">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="stateLabel4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="stateLabel5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="stateLabel6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="stateLabel7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="stateLabel8">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="modelLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="topWidget1" native="true"/>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="topWidget2" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="settingComboBox3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TOFD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
